tests/test_cli_files.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA client files tests."""
+
+import hashlib
+import io
```

Comment:
Not sure if this is wip, commenting on a few lines that popped up on my editor:
io is unused

Reply:
Plugin that tracks unused variables/imports stoped working. Thanks for notice.